pubspec.yaml

```diff
@@ -1,7 +1,6 @@
 name: clip_regular_polygon
 description: A new Flutter package project.
 version: 0.0.1
-homepage:
```

Comment:
Also, if you are already modifying the `pubspec.yaml`, write a proper `description` and add
```yaml
repository: https://github.com/JimGerth/clip_regular_polygon-flutter
issue_tracker: https://github.com/JimGerth/clip_regular_polygon-flutter/issues
```

Reply:
Done.